test/e2e/build_test.go

```diff
@@ -441,24 +441,22 @@ RUN find /test`, CITEST_IMAGE)
 	It("podman remote build must not allow symlink for ignore files", func() {
 		// Create a random file where symlink must be resolved
 		// but build should not be able to access it.
-		privateFile := filepath.Join("/tmp", "private_file")
+		privateFile := filepath.Join(podmanTest.TempDir, "private_file")
 		f, err := os.Create(privateFile)
 		Expect(err).ToNot(HaveOccurred())
 		// Mark hello to be ignored in outerfile, but it should not be ignored.
 		_, err = f.WriteString("hello\n")
 		Expect(err).ToNot(HaveOccurred())
 		defer f.Close()
 
-		// Create .dockerignore which is a symlink to /tmp/private_file.
+		// Create .dockerignore which is a symlink to /tmp/.../private_file outside of the context dir.
```

Comment:
Touching files in cwd is EVIL. Even if my package-e2e PR doesn't merge, I will be insisting on cleaning up this horrible practice: 
https://github.com/containers/podman/pull/22395/files#diff-8b4d34c763f953caff8c6747e966aa1a7e869411747cd000370918887ba5b124

Reply:
yeah agreed, I didn't look that closely at the test just the leaked files.
FYI if you wonder where the `oom` file is created in the cwd when you run all tests see https://github.com/containers/conmon/issues/504